app/javascript/components/UI/ShoutoutButton.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+const ShoutoutButton = () => {
+  return (
+    <div style={{position: 'fixed', left: 45, top:756 }} >
+        <svg width="102" height="103" viewBox="0 0 102 103" fill="none" xmlns="http://www.w3.org/2000/svg" xmlnsXlink="http://www.w3.org/1999/xlink">
```

Comment:
Create/add  pls new SVG file to assets/images folder and use it here.

Reply:
Done